library/geonetwork

```diff
@@ -13,10 +13,18 @@ Tags: 3.6.0-postgres, 3.6-postgres
 GitCommit: 9a28f0a61a8b043b1562f79caed9fdec3c009452
 Directory: 3.6.0/postgres
 
-Tags: 3.8.0, 3.8, latest
+Tags: 3.8.0
 GitCommit: e946f10156ed14097351c05cb79349dfda046c0d
 Directory: 3.8.0
 
-Tags: 3.8.0-postgres, 3.8-postgres, postgres
+Tags: 3.8.0-postgres
 GitCommit: e946f10156ed14097351c05cb79349dfda046c0d
 Directory: 3.8.0/postgres
```

Comment:
Any particular reason to keep this version listed here?  Removing tags here will remove them from the "Supported" section on the Hub readme (and will prevent us from spending cycles rebuilding them on the official build servers), but the tags will still be available to users who want them.  (See https://github.com/docker-library/official-images#library-definition-files for more detail on this.)

Reply:
`3.8.1` is a patch release that add bug and security fixes to `3.8.0` release. Indeed we can remove that tag from the list.

Reply:
I've removed `3.8.0` version.